test/logger/server.js

```diff
@@ -140,6 +141,33 @@ test('logging methods should generate any output when of an adequate level', t =
 	process.stdout.write.restore();
 });
 
+test('should not output an empty object', t => {
```

Comment:
There's times where it can be useful to know that something is an empty object. This would hide that from the logs.

Reply:
I see, I should check if metadata is `undefined` then.

Reply:
Unfortunately, it's impossible to do something like that. The `meta` will be always an object whatever the case.

So I have two options:
- Write the meta in the console and file transport
- Only write the `meta` object in the log file, and hide it on console

Which one would be better for you?

Reply:
Or maybe it's possible by moving the serialization to the `Logger` class methods. I'll try this way.